src/QMCDrivers/QMCLinearOptimize.cpp

```diff
@@ -40,6 +40,22 @@
 
 namespace qmcplusplus
 {
+enum LinearOptTimers
```

Comment:
Currently all the timer details are not in the header file.   It seems preferable to keep it that way unless there is a strong reason to do otherwise.

Reply:
Could you explain me why you think the current way is preferable?
I didn't mean just move to the header but move into the class. In the current way, I cannot use a timer in a function defined in the header.

Reply:
Yes, that's a good point.  More stuff in the headers means more recompilation if something changes (though the timers probably don't change much)
I'm wondering if we should drop the myTimer list and the enums.   As seen in this diff, it was an attempt to clean up existing practice.   But it adds code overhead.   Maybe we should just declare the timers directly and initialize in the constructor, as is done in the Fermion classes.